.github/workflows/ci.yml

```diff
@@ -31,6 +31,18 @@ jobs:
           override: true
       - run: cargo build --target ${{ matrix.target }} --release --no-default-features
 
+  # Mimics the setup of docs.rs, but fails on warnings
+  build-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
```

Comment:
Why build docs in nightly?

Reply:
That's how docs.rs does it. In particular, needed for `doc_cfg` and `doc_auto_cfg` features.